Resources/config/kernel.yml

```diff
@@ -0,0 +1,8 @@
+system:
+    default:
+        content_view:
+            line:
+                pjax_location_line:
+                    template: "eZPlatformUIBundle:content:line.html.twig"
```

Comment:
Given what this template does, I'd specialize it's name a little bit. Maybe `pjax_line.html.twig` ?